.gitignore

```
@@ -3,6 +3,4 @@
 
 # CMake
 cmake-build-debug/
-
-
-
+/build/Debug/resources
```

I would change this to this and redo the commit:
```suggestion
build/
```
We don't need all the build files.